kong/pdk/log.lua

```diff
@@ -826,37 +841,20 @@ do
 
   else
     function serialize(options)
-      check_phase(PHASES_LOG)
-
-      local ongx = (options or {}).ngx or ngx
-      local okong = (options or {}).kong or kong
+      options = options or {}
+      local ongx = options.ngx or ngx
+      local okong = options.kong or kong
+      
+      local authenticated_entity, tls_info = prepare_serialize(ongx, okong)
```

Comment:
should be `prepare_serialize(ongx.ctx, ongx.var)`

Reply:
fixed